react/features/base/config/configWhitelist.ts

```diff
@@ -195,7 +195,6 @@ export default [
     'p2p',
     'participantsPane',
     'pcStatsInterval',
-    'preferredCodec',
```

Comment:
We need to keep backwards compat, so this still needs to be whitelisted.

Reply:
The `videoQuality` and `p2p` sections are whitelisted so we don't need `preferredCodec` to be whitelisted for the deprecated settings to work. I think it was added by mistake.